src/vulkan/device.cc

```diff
@@ -610,6 +614,10 @@ Result Device::Initialize(
       return amber::Result(
           "Subgroup extended types requested but feature not returned");
     }
+    if (feature == kAccelerationStructure &&
```

Comment:
This seems unrelated?

Reply:
yes, I think it's some merge gone bad. I'll re-do it